public/app/stores/roomStore.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var React = require('react');
+var Reflux = require('reflux');
+
+var roomActions = require('../actions/roomActions');
+
+// TODO: Is this the correct naming style for global variables?
+var _rooms = [
+    {title: 'Home', devices: [
+        {title: 'yetu Home Gateway', type: 'Home Gateway', connected: true},
+        {title: 'Nest', type: 'Thermostat', connected: true}
+    ]},
+    {title: 'Living room', devices: [
+        {title: 'Nest', type: 'Thermostat', connected: false}
+    ]},
+    {title: 'Bed room', devices: []}
+];
+
+var roomStore = Reflux.createStore({
+    listenables: [roomActions],
+    
+    getRooms: function() {
+        // We don't want callers to modify the array, thus we return a fresh copy
+        return _rooms.slice();
+    },
+    
+    onCreateRoom: function() {
+        console.log('onCreateRoom');
+        var room = {
+            title: 'New Room',
+            devices: []
+        };
+        this.updateRooms(_rooms.concat(room));
+    },
+
+    updateRooms: function(rooms) {
+        _rooms = rooms;
+        // Send the updated rooms to all listening components
+        this.trigger(rooms);
+    }
+});
+
+module.exports = roomStore;
```

Comment:
If you are using modules, then your "global variables" are not global - they are scoped by the module. They are "module-local"